src/main/scala/com/cognite/sdk/scala/v1/functions.scala

```diff
@@ -102,3 +102,5 @@ final case class FunctionScheduleCreate(
     cronExpression: String = "",
     data: Option[Json] = None
 )
+
+final case class FunctionCallData(data: Json, nonce: String)
```

Comment:
`FunctionCallResponse` is used for some another endpoint. Therefore using `FunctionCallRequest` could be confusing.